pom.xml

```diff
@@ -92,7 +92,13 @@
 			<artifactId>commons-lang3</artifactId>
 			<version>${commons.lang3.version}</version>
 		</dependency>
-	</dependencies>
```

Comment:
Indent looks wrong.
</dependencies> shouldn't have moved and the added indent should line up with others at the same level.

Reply:
Done